pkg/disk/json_test.go

```diff
@@ -0,0 +1,127 @@
+package disk
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/rsa"
+	"fmt"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/go-jose/go-jose/v3"
+	"github.com/go-jose/go-jose/v3/cryptosigner"
+	"github.com/go-jose/go-jose/v3/jwt"
+	"github.com/spiffe/go-spiffe/v2/bundle/jwtbundle"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/go-spiffe/v2/svid/jwtsvid"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	jwtBundleFilename = "jwt_bundle.json"
+	jwtSVIDFilename   = "jwt.json"
+)
+
+func TestWriteJWTBundleSet(t *testing.T) {
+	td, err := spiffeid.TrustDomainFromString("example.test")
+	require.NoError(t, err)
+
+	jwtBundle := jwtbundle.New(td)
+	jwtBundleSet := jwtbundle.NewSet(jwtBundle)
+	require.True(t, jwtBundleSet.Has(td))
+
+	tempDir := t.TempDir()
+
+	err = WriteJWTBundleSet(jwtBundleSet, tempDir, jwtBundleFilename)
+	require.NoError(t, err)
+
+	actualJWTBundle, err := jwtbundle.Load(td, path.Join(tempDir, jwtBundleFilename))
+	require.NoError(t, err)
+	require.Equal(t, jwtBundle, actualJWTBundle)
+}
+
+func TestWriteJWTSVID(t *testing.T) {
+	spiffeID, err := spiffeid.FromString("spiffe://example.test/workload")
```

Comment:
NIT: here you can use  spiffeid.RequireFromString

Reply:
done